cogs/afterhours/afterhours.py

```diff
@@ -265,3 +420,245 @@ async def afterHoursSet(self, ctx: Context):
         """Set the channel for notifications."""
         await self.config.guild(ctx.guild).get_attr(KEY_CTX_CHANNEL_ID).set(ctx.channel.id)
         await ctx.send("Using this channel to construct context later!")
+
+    @checks.mod()
+    @afterHours.group(name="autopurge")
+    async def afterHoursAutoPurge(self, ctx: Context):
+        """Manage auto-purge
+
+        Auto-purge is by default executed at the same frequency as the AfterHours background loop.
+        Refer to the subcommands section to see the subcommand to disable this behavior.
+        """
+
+    @checks.mod()
+    @afterHoursAutoPurge.command(name="now")
+    async def afterHoursAutoPurgeNow(self, ctx: Context):
+        """Execute auto-purge immediately"""
+        await ctx.send("Purging...")
+        await self.doAutoPurge(forced=True)
+        await ctx.send("Purge completed!")
+
+    @checks.admin()
+    @afterHoursAutoPurge.command(name="togglebackground")
+    async def afterHoursAutoPurgeToggleBackgroundLoop(self, ctx: Context):
+        """Toggle auto-purge background loop
+        If it is disabled, the AfterHours background loop will not execute auto-purge.
+        """
+        guildConfig = self.config.guild(ctx.guild)
+        autoPurgeConfig = guildConfig.get_attr(KEY_AUTO_PURGE)
+        bgLoopConfig = autoPurgeConfig.get_attr(KEY_BACKGROUND_LOOP)
+        if await bgLoopConfig() is True:
+            await bgLoopConfig.set(False)
+            await ctx.send("Disabled auto-purge background loop")
+        else:
+            await bgLoopConfig.set(True)
+            await ctx.send("Enabled auto-purge background loop")
+
+    @checks.admin()
+    @afterHoursAutoPurge.command(name="inactiveduration")
+    async def afterHoursAutoPurgeInactiveDuration(self, ctx: Context, duration: Optional[str]):
```

Comment:
I think you can remove the space requirement by doing the following, since duration is the last and only parameter:
```suggestion
    async def afterHoursAutoPurgeInactiveDuration(self, ctx: Context, *, duration: Optional[str]):
```

Reply:
Tested and it worked. Thanks for the tip.